docusaurus/docs/Android/04-compose/03-channel-components/02-channel-list-header.mdx

```diff
@@ -52,12 +52,10 @@ fun ChannelListHeader(
 ```
 
 * `onAvatarClick`: Handler when the user taps on their own avatar.
-* `onHeaderActionClick`: Handler when the user taps on the trailing action. This is only used if you don't override the `action` parameter, which defines the UI for the trailing icon.
-
-:::note 
-
-The `action` parameter uses the `onHeaderActionClick` parameter for the default header action. If you want to keep the same UI but override the behavior, you need to change `onHeaderActionClick`. Otherwise, skip over to **Customization** to learn how to customize the action UI.
```

Comment:
We should hyperlink the word 'Customization' so that it points to the appropriate section. 

The section is almost visible from that part but I think it would still be good to link it for the sake of consistency.